taskcluster/kinds/integration-test/kind.yml

```diff
@@ -23,3 +23,18 @@ tasks:
       test_platform:
         - android-hw
         - macosx
+
+  translations:
+    description: "Run translations pipeline integration tests"
+    decision-index-paths:
+      # TODO: switch to production translations repo
```

Comment:
Did you mean to switch this prior to landing?

Reply:
Sorry, I forgot to comment on this! Yes, this will be switched. It's waiting on https://github.com/mozilla-releng/fxci-config/pull/228, https://github.com/mozilla/translations/pull/966, and a cron run of the pipeline once those land.